pages/persons/view/person_current_view_page.py

```diff
@@ -0,0 +1,229 @@
+# coding=utf-8
+from selenium.webdriver.common.by import By
+from selenium.webdriver.firefox import webdriver
+from pages.persons.view.person_main_view_page import PersonMainViewPage
+
+__author__ = 'Administrator'
+
+class PersonCurrentViewPage(PersonMainViewPage):
+
+    # >>>>>>>>>>>>>>>>>>>>>>>>>>> Global locators for information about person <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
+    FIO_PERSON_UKRANIAN = (By.XPATH, "//h2[@class='text-primary ng-binding']")
+    FIO_PERSON_ENGLISH = (By.XPATH, "//h4[@class='ng-binding']")
+
+    SERIAL_PERSONAL_RECORD = (By.XPATH, "//p[contains(.,'Серія')]")
+    NUMBER_PERSONAL_RECORD = (By.XPATH, "//p[contains(.,'Номер')]")
+
+    # >>>>>>>>>>>>>>>>>>>>>>>>>>> Main locators for information about person <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
+    # This titles for use in tests
+    arr_titles_main = ["TYPE_OF_PERSON", "DATE_OF_BIRTH", "RESIDENT", "SEX", "MARITAL_STATUS", "MILITARY_SERVICE",
+                      "CITIZENSHIP", "INDEX_MATERIAL_RESPONSIBLE", "HOSTEL"]
+    TYPE_OF_PERSON = (By.XPATH, "//p[contains(.,'Тип персони')]")
+    DATE_OF_BIRTH = (By.XPATH, "//p[contains(.,'Дата народження')]")
+    RESIDENT = (By.XPATH, "//p[contains(.,'Резидент')]")
+    SEX = (By.XPATH, "//p[contains(.,'Стать')]")
+    MARITAL_STATUS = (By.XPATH, "//p[contains(.,'Сімейний стан')]")
+    MILITARY_SERVICE = (By.XPATH, "//p[contains(.,'Військовозобовязаний')]")
+    CITIZENSHIP = (By.XPATH, "//p[contains(.,'Громадянство')]")
+    INDEX_MATERIAL_RESPONSIBLE = (By.XPATH, "//p[contains(.,'Індекс матеріально відповідального')]")
+    HOSTEL = (By.XPATH, "//p[contains(.,'Гуртожиток')]")
+
+    # Date of birth
+    arr_titles_birth = ["PLACE_OF_BIRTH_LEVEL_1", "PLACE_OF_BIRTH_LEVEL_2", "PLACE_OF_BIRTH_LEVEL_3",
+                "PLACE_OF_BIRTH_LEVEL_4", "PLACE_OF_BIRTH_LEVEL_5", "PLACE_OF_BIRTH_LEVEL_6", "PLACE_OF_BIRTH_LEVEL_7"]
+    FOR_COUNT_ELEMENTS_DATE_OF_BIRTH = (By.CSS_SELECTOR, ".form-group.col-md-10>label+div .form-group.ng-scope")
+    PLACE_OF_BIRTH_LEVEL_1 = (By.CSS_SELECTOR, ".form-group.col-md-10>label+div>div:nth-child(1) span:nth-child(2) span")
```

Comment:
can we just write ONE method that returns that 7 tuples (method should accept number 1-7 as a parameter)